src/js/tubular/interceptors/no-cache.js

```diff
@@ -15,12 +15,13 @@
 
             return {
                 request: (config) => {
-                    if (config.method === 'GET' &&
-                    config.url.indexOf('.htm') === -1 &&
-                    config.url.indexOf('blob:') === -1 &&
-                    config.url.indexOf('noCache=') === -1) {
+
+                    // patterns to escape: .htm | blob: | noCache=
+                    const matchesEspacePatterns = (/\.htm|blob:|noCache\=/.test(config.url));
```

Comment:
I would assign config.url to a local var in order to avoid repeating of config.url